Compose/BottomStickyButtonScaffold/src/main/kotlin/com/infomaniak/core/compose/bottomstickybuttonscaffold/BottomStickyButtonScaffold.kt

```diff
@@ -0,0 +1,122 @@
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.core.compose.bottomstickybuttonscaffold
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.BoxScope
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.CenterAlignedTopAppBar
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.SnackbarHost
+import androidx.compose.material3.SnackbarHostState
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBarDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.infomaniak.core.compose.basicbutton.BasicButton
+import com.infomaniak.core.compose.margin.Margin
+import com.infomaniak.core.compose.preview.PreviewLargeWindow
+import com.infomaniak.core.compose.preview.PreviewSmallWindow
+
+private val DEFAULT_SINGLE_PANE_MAX_WIDTH = 800.dp
+
+@Composable
+fun BottomStickyButtonScaffold(
+    modifier: Modifier = Modifier,
+    snackbarHostState: SnackbarHostState? = null,
+    maxScreenWidth: Dp = DEFAULT_SINGLE_PANE_MAX_WIDTH,
+    buttonComboVerticalPadding: Dp = Margin.Small,
+    topBar: @Composable () -> Unit,
+    topButton: @Composable ((Modifier) -> Unit)? = null,
+    bottomButton: @Composable ((Modifier) -> Unit)? = null,
+    content: @Composable BoxScope.() -> Unit,
+) {
+    SinglePaneScaffold(
+        modifier = modifier,
+        maxScreenWidth = maxScreenWidth,
+        snackbarHost = { snackbarHostState?.let { SnackbarHost(hostState = it) } },
+        topBar = topBar,
+    ) { contentPaddings ->
+        Column(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(contentPaddings),
+            horizontalAlignment = Alignment.CenterHorizontally,
+        ) {
+            Box(modifier = Modifier.weight(1.0f), content = content)
+            DoubleButtonCombo(
+                modifier = Modifier.padding(vertical = buttonComboVerticalPadding),
+                doubleButtonMaxWidth = maxScreenWidth,
```

Comment:
DoubleButtonCombo should not need to precise that its max width applies to "doubleButton", it can hardly apply to anything else

```suggestion
                maxWidth = maxScreenWidth,
```